Library/Formula/cvc4.rb

```diff
@@ -0,0 +1,40 @@
+class Cvc4 < Formula
+  homepage "http://cvc4.cs.nyu.edu/"
+  url "http://cvc4.cs.nyu.edu/builds/src/cvc4-1.4.tar.gz"
+
+  devel do
+    url "http://cvc4.cs.nyu.edu/builds/src/unstable/latest-unstable.tar.gz"
```

Comment:
We actually do use the new features quite a bit for development work, though not for everyday use.